src/pages/item/item-actions/itemActionsButtons/loadRelationsButton/loadRelationsButton.tsx

```diff
@@ -0,0 +1,191 @@
+import React, { useState } from "react";
+import { createConnectorsForDownstreamRefsAndAssociation } from "../../../../../api/miro.api";
+import getAppCardIds from "../../../../../api/utils/getAppCardIds";
+import { useItemRelations } from "../../../../../hooks/useItemRelations";
+import doAllConnectorsExist from "../../../../../api/utils/doAllConnectorsExist";
+import removeConnectors from "../../../../../api/utils/removeConnectors";
+
+interface Association {
+  associationId: number;
+  targetItemId: number;
+}
+
+export default function LoadRelationsButton(props: {
```

Comment:
There's about 5 compiler issues right now.  
They're not just there to be ignored, and I'm convinced you can eliminate all of them.  

The only compiler errors we can generally just ignore right now are those about the miro sdk types, because they just didn't properly keep them in line with their actual features. Everything else should be taken seriously and addressed adequately.